### PR TITLE
Added support for args

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

**Comment by brian-avery** [comment]
Based on conversation earlier, the repos in here need to change to new repos.

**Review by dmitri-d — APPROVED** [review]

File: istio-proxy/istio-proxy.spec [hunk 1]
```diff
@@ -143,6 +137,10 @@ TEST_ENVOY=false RUN_TESTS=true %{SOURCE2}
 /usr/local/bin/envoy
 
 %changelog
+* Tue Feb 11 2020 Brian Avery <bavery@redhat.com> - 1.1.0-1
+- Updated to Istio 1.4 proxy
+- Added support for building based on SHA
+- Simplified repositories
```
Comment (dgn) [inline istio-proxy/istio-proxy.spec:143]: I love it! Was hoping we'd eventually get a Maistra/envoy repo

**Review by dgn — COMMENTED** [review]
Looks good, but I can't get it to build... This seems to depend on a local  OpenSSL lib which somehow bazel can't find. It fails at the fetch step already

**Comment by brian-avery** [comment]
@dgn

What version of Bazel are you on?
Are you referring to update.sh or the build itself? 
What errors are you seeing?

**Comment by dgn** [comment]
> What version of Bazel are you on?

Using bazelisk, so whatever version we need (we set it here to 1.1.0)
> Are you referring to update.sh or the build itself?

I'm running update.sh. I don't even get to the build, it fails in fetch.sh.

> What errors are you seeing?

Lost my terminal history but basically, bazel coudn't find openssl. This didn't fail earlier, when we were replacing boringssl with openssl after the fetch.

Environment problems maybe? This is on Fedora 31.

**Review by dmitri-d — APPROVED** [review]

**Review by jwendell — APPROVED** [review]

**Comment by brian-avery** [comment]
@Mergifyio refresh

**Pull request merged**